### PR TITLE
Collab fixes4

### PR DESCRIPTION
**Pull request by dmulfinger**
These changes were requested by Hemil, Brandon, and Edgar.  Some required fields shouldn't be required.  Also some required field names were incorrect.

### CODE REVIEW

File: TCL4 Data Management/utm-tcl4-dmp-con.yaml [hunk 1]
```diff
@@ -100,9 +100,6 @@ definitions:
     type: object
     required:
       - "metaData"
-      - "plannedBvlosLandingPoints"
-      - "plannedVlosLandingPoints"
```
Comment (akishiha) [inline TCL4 Data Management/utm-tcl4-dmp-con.yaml]: Why are we making this optional?  thanks

Comment (dmulfinger) [reply TCL4 Data Management/utm-tcl4-dmp-con.yaml]: This is ConPreRunOp, so this should be submitted per op. Some ops may not have BVLOS landing points and others may not have VLOS landing points.  (according to Hemil, Edgard, and Brandon).

File: TCL4 Data Management/utm-tcl4-dmp-con.yaml [hunk 2]
```diff
@@ -181,10 +178,6 @@ definitions:
     required:
       - "metaData"
       - "timestamp"
-      - "temperature"
-      - "pressure"
-      - "windSpeed"
```
Comment (akishiha) [inline TCL4 Data Management/utm-tcl4-dmp-con.yaml]: these should be mandatory otherwise you can have an object with zero data

Comment (nasajoey) [reply TCL4 Data Management/utm-tcl4-dmp-con.yaml]: The problem is that not every sensor will have all those measurements.

Comment (akishiha) [reply TCL4 Data Management/utm-tcl4-dmp-con.yaml]: ok

**Comment by akishiha** [comment]
@dmulfinger go ahead and merge into develop.  Then you can run ./update_references.sh.  It will prompt you for a branch.   Enter develop.  Commit those changes as well.

**Comment by dmulfinger** [comment]
> @dmulfinger go ahead and merge into develop. Then you can run ./update_references.sh. It will prompt you for a branch. Enter develop. Commit those changes as welI.

I ran the update_references.sh and it did not appear to make any changes.  Everything seems to point to develop already. ?

**Comment by akishiha** [comment]
no change is fine - thanks

**Pull request merged**